rollup.config.js

```diff
@@ -84,14 +84,12 @@ export default [
       { file: pkg.main, format: 'cjs', sourcemap: true },
       { file: pkg.module, format: 'es', sourcemap: true }
     ],
-    plugins: [
+    plugins: defaultBrowserPluginOptions.concat([
```

Comment:
@viviivanov in the defaultBrowserPluginOptions is the transpilation task defined. See line 21 for it.